acceptance/acceptance_test.go

```diff
@@ -285,6 +290,26 @@ func testAccept(t *testing.T, inprocessMode bool, singleTest string) int {
 				testutil.WriteFile(t, filepath.Join(dir, internal.MaterializedConfigFile), materializedConfig)
 			}
 
+			t.Cleanup(func() {
+				if skipReason != "" {
+					skippedTestCount += 1
+					return
+				}
+				completedTestCount += 1
+
+				elapsed := time.Since(startTime)
+				if t.Failed() {
+					t.Logf("[progress] Test %s failed in %s", dir, elapsed.Round(time.Second))
+					return
+				}
+				if elapsed > time.Minute {
+					t.Logf("[progress] Test %s is a long running test. It took %s to complete", dir, elapsed.Round(time.Second))
```

Comment:
You have "make slowest" to show top slowest tests in the last run.

you also have test-output.json for getting this and other info.